src/services/transcript.ts

```diff
@@ -18,6 +18,26 @@ export type TranscriptItems = {
   activities: MembershipHistory[];
 };
 
+export type Groupexperience = {
+  Job_Title: string;
+  job: StudentEmployment[] | undefined;
+  latestDate?: string;
+};
+
+export type NewStudentEmployment = {
```

Comment:
This needs to become StudentEmployment, and the old one above should be deleted.